visualizer.py

```diff
@@ -0,0 +1,311 @@
+#!/usr/bin/python3
+"""
+Author: Vertigo Designs, Ryan Summers
+
+Description: Bokeh application for serving Stabilizer stream visuals.
+"""
+import argparse
+import asyncio
+import enum
+import logging
+import socket
+from typing import List, Mapping
+
+import bokeh.plotting
+import bokeh.layouts
+import bokeh.document
+import bokeh.palettes
+import bokeh.io
+import bokeh.models
+import bokeh.server.server
+import numpy as np
+
+from miniconf import Miniconf
+from stabilizer.stream import StabilizerStream
+
+# The sample rate of stabilizer.
+SAMPLE_RATE_HZ = 100e6 / 128
+
+
+def _get_ip(broker):
+    """ Get the IP of the local device.
```

Comment:
The docs I write are largely just habit and don't take long. I prefer to take the approach of documenting everything as opposed to putting it off because I find that I rarely return to code to document it.

Reply:
Then it needs to be high quality documentation that is concise and accurate. Otherwise reviewing takes a lot of time. Also there is a significant effort multiplier due to likely code changes.